addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -21,7 +21,7 @@
                         'o-large': props.imageSize === 'large',
                         'o-medium': props.imageSize === 'medium',
                         'o-small': props.imageSize === 'small',
-                    }" t-att-href="attachmentUrl" t-att-style="imageStyle" t-att-data-mimetype="attachment.mimetype"
+                    }" t-att-href="attachmentUrl" t-att-style="imageStyle" role="menuitem" aria-label="View image" t-att-tabindex="attachment.isViewable ? 0 : -1" t-att-aria-disabled="!attachment.isViewable" t-att-data-mimetype="attachment.mimetype"
```

Comment:
Is it not a problem to set `role="menuitem"` even when the attachment is not actually clickable?
Same for `aria-label="View image"`, it looks wrong to have this text when the attachment is not even an image.

Why do we bother setting all those tags? If we remove them when not applicable then we don't need to bother with aria-disabled either? It's never going to get enabled on a particular attachment if it wasn't initially.

Reply:
I think it's not a problem. Since it's not focusable and it's disabled, it's ignored by assistive technologies anyway.

The ideal solution would be to use `aria-idden="true"`, but, unfortunately, that attribute causes also styling changes (it has CSS meaning, which I think it's not correct BTW).